.github/CONTRIBUTING.md

```diff
@@ -87,6 +88,7 @@ Things you **CAN** do:
 
 Things you **CAN'T** do:
 * [Close PRs](https://imgur.com/w2RqpX8.png): Only maintainers are allowed to close PRs. Do not hit that button.
+* Close issues purely for breaking a template if the same information is contained without it.
```

Comment:
Okay but if something is conveyed in an extremely difficult to parse manner through the markdown text editor, yet still contains the relevant information, does this still allow for reformatting a pr/issue to make it more legible? What about attempts to revert a change to improve legibility back to the more illegible state, if a change is permitted?